app/views/shared/_accessibility_features.html.erb

```diff
@@ -35,7 +35,7 @@
         <tr>
             <td style="width:300px;"> Toggle Game Casing </td>
             <td class="user-options"> 
-                <button style="background-color:transparent; border:none;" onclick="changeGameCasing()">
+                <button data-turbo="false" style="background-color:transparent; border:none;" onclick="changeGameCasing()">
```

Comment:
@krishnacalindi is this how this is supposed to be used?

Reply:
Yes it is. Hope this fixes it.